Controller/ContentManagement/CrudController.php

```diff
@@ -358,4 +361,46 @@ public function getContentTypeInfo(ContentType $contentType)
                 'contentType' => $contentType,
         ]);
     }
+
+    /**
+     * @Route("/api/user-profile", defaults={"_format": "json"}, methods={"GET"})
+     */
+    public function getUserProfile() : JsonResponse
+    {
+        $user = $this->getUser();
+        if (! $user instanceof User) {
+            throw new \RuntimeException('User profile class not recognized');
+        }
+        if (! $user->isEnabled()) {
+            throw new \RuntimeException('User disabled');
+        }
+
+        return $this->json([
+            'username' => $user->getUsername(),
+            'displayName' => $user->getDisplayName(),
+            'roles' => $user->getRoles(),
+            'email' => $user->getEmail(),
+            'circles' => $user->getCircles(),
+        ]);
+    }
+
+    /**
+     * @Route("/api/user-profiles", defaults={"_format": "json"}, methods={"GET"})
+     */
+    public function getUserProfiles(UserService $userService) : JsonResponse
```

Comment:
Question : peut-on "autowirer" un service dans un bundle ?

Reply:
Non c'est vrai. J'ai été trop vite

Reply:
@Davidmattei what do you think about Jérome's question? How should we attach services to controller in the core bundle

Reply:
Inject them in the constructor, autowiring in third party bundles should not be done. It makes overwriting them harder.

Services should not use autowiring or autoconfiguration. Instead, all services should be defined explicitly._

https://symfony.com/doc/current/bundles/best_practices.html#services

Reply:
Fixed cfe5cdb5f353154596abff458eebe79179ee0cec